### PR TITLE
Release notes datahub 1.15

### PR DESCRIPTION
**Pull request by kdp-cloud**
- Add ISA compliance features to release notes
- Update docker compose syntax to version 2

### CODE REVIEW

**Review by stuzart — APPROVED** [review]
thanks for docker compose updates, it was something on my todo list :)

File: tech/releases/index.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ Release date: _March 2024_
 * Fix to LS Login authentication configuration
 * Support for generic OpenID Connect authentication to be configured
 * Extended Metadata write API, and documentation for both read and write.
+* Single Page view for visualising and creating ISA-JSON compliant experiments.
```
Comment (floradanna) [inline tech/releases/index.md:40]:
```suggestion
* Single Page view for visualising experiments.
```

File: tech/releases/index.md [hunk 2]
```diff
@@ -37,6 +37,8 @@ Release date: _March 2024_
...
+* Usage of Sample Type Templates to help organise metadata in an ISA-JSON compliant manner.
```
Comment (floradanna) [inline tech/releases/index.md:41]:
```suggestion
* Creation of ISA-JSON compliant experiments using Experiment Sample Templates.
```

**Review by floradanna — COMMENTED** [review]
I have provided some changes. Please include them in the release note.

**Pull request merged**